NIPTool/commands/load/base.py

```diff
@@ -10,6 +10,7 @@
 
 # commands
 from NIPTool.commands.load.batch import batch as batch_command
```

Comment:
I'm wondering: if in the future you add other batch-related command how would you name them in this file?
 Perhaps assign it a more self-explicative name, something like this:
`from NIPTool.commands.load.batch import batch as load_batch_cmd`

Reply:
good point